common/src/main/java/org/apache/drill/common/collections/MapWithOrdinal.java

```diff
@@ -54,7 +51,7 @@
 public class MapWithOrdinal<K, V> implements Map<K, V> {
   private final static Logger logger = LoggerFactory.getLogger(MapWithOrdinal.class);
 
-  private final Map<K, Entry<Integer, V>> primary = Maps.newLinkedHashMap();
+  private final Map<K, Entry<Integer, V>> primary = new LinkedHashMap<>();
   private final IntObjectHashMap<V> secondary = new IntObjectHashMap<>();
 
   private final Map<K, V> delegate = new Map<K, V>() {
```

Comment:
@vvysotskyi The same question as in #1264 what is the purpose (use case) of the class?